packages/components/src/components/SquiggleViewer/utils.ts

```diff
@@ -17,11 +19,24 @@ export function pathAsString(path: SqValuePath) {
   return path.items.join(".");
 }
 
+export const pathItemFormat = (item: PathItem): string => {
+  if (item.type === "cellAddress") {
+    return `Cell (${item.value.row},${item.value.column})`;
```

Comment:
`TableCellsIcon` might be nice here.

Reply:
I briefly tried this, wasn't sure how I felt about it. Can bring it back, if tables are popular.